README.markdown

```diff
@@ -88,8 +88,8 @@ TABLE: weak_guy;
 +---------------+-------------+------+-----+---------+----------------+
 | Field         | Type        | Null | Key | Default | Extra          |
 +---------------+-------------+------+-----+---------+----------------+
-| strong_guy_id | int(10)     | NO   | PRI | 0       |                |
-| weak_guy_id   | int(10)     | NO   | PRI | NULL    | auto_increment |
+| strong_guy_id | int(10)     | NO   |     | 0       |                |
```

Comment:
No, the primary key is made up of both. The field weak_guy_id is auto
increment.

It's correct as it is. Are you not familiar with composite primary keys?
On Sep 4, 2015 6:46 PM, "Dumitru Uzun" <notifications@github.com> wrote:

> In README.markdown
> <https://github.com/iaindooley/PluSQL/pull/2#discussion_r38732234>:
>
> > @@ -88,8 +88,8 @@ TABLE: weak_guy;
> >  +---------------+-------------+------+-----+---------+----------------+
> >  | Field         | Type        | Null | Key | Default | Extra          |
> >  +---------------+-------------+------+-----+---------+----------------+
> > -| strong_guy_id | int(10)     | NO   | PRI | 0       |                |
> > -| weak_guy_id   | int(10)     | NO   | PRI | NULL    | auto_increment |
> > +| strong_guy_id | int(10)     | NO   |     | 0       |                |
>
> One of two has to be changed:
> 1. strong_guy_id is not PRI
> 2. weak_guy_id is not auto_increment
>
> I just guessed it is 1., but change it to 2.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iaindooley/PluSQL/pull/2/files#r38732234>.
>